xl/common.py

```diff
@@ -294,9 +308,9 @@ def wrapper(*args, **kwargs):
         stats = pstats.Stats(prof)
         stats.strip_dirs()
         stats.sort_stats('time', 'calls')
-        print ">>>---- Begin profiling print"
+        print(">>>---- Begin profiling print")
```

Comment:
Not actually needed unless you are printing multiple things, e.g. `print(1, 2)` will print `(1, 2)` otherwise.

But in general I try to code with `from __future__ import division, print_function, unicode_literals`.